fairscale/utils/reduce_scatter_bucketer.py

```diff
@@ -35,11 +35,17 @@ def flush(self) -> None:
         # reduce-scatter bucket
         if hasattr(dist, "_reduce_scatter_base") and enable_nccl_base_collectives:
             dist._reduce_scatter_base(
-                self.output_shard[: self.offset], self.data[:, : self.offset].contiguous(), group=self.group
+                self.output_shard[: self.offset],
+                self.data[:, : self.offset].contiguous(),
+                group=self.group,
+                async_op=False,
```

Comment:
similar to the comment above, let's use a variable instead of True/False values in multiple places in this file.